modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/provider/zoom/ZoomPowwowServiceProvider.java

```diff
@@ -428,20 +354,19 @@ protected JSONObject execute(
 		}
 
 		String location = sb.toString();
-
 		options.setLocation(location);
 
-		Map<String, String> parts = new HashMap<>();
-
-		if (parameterMap != null) {
-			parts.putAll(parameterMap);
-		}
+		String token = getToken(powwowServer);
 
-		options.setParts(parts);
+		Map<String, String> headers = new HashMap<>();
+		headers.put("Authorization", "Bearer " + token);
+		headers.put("Content-Type", "application/json");
```

Comment:
just curious, not mean you have to change it right now, is that possible if we do `options.addHeader("Content-Type", "application/json");`?